features/step_definitions/content_steps.rb

```diff
@@ -23,6 +23,17 @@
     fail unless page.has_content?(arg1)
   end
 end
+
+#
+# Test for a text in the whole page
+#
+Then(/^I should see a "([^"]*)" or "([^"]*)" text$/) do |text, text2|
+  unless page.has_content?(text) || page.has_content?(text2)
+    sleep 25
```

Comment:
I would remove this sleep from here, as we don't want to wait in case we are reusing this step in other feature. I'd put a "wait step" in the above scenario 😄